src/System.Net.Http/src/uap/System/Net/cookie.cs

```diff
@@ -85,7 +85,7 @@ public sealed class Cookie
         int[] m_port_list = null;
         bool m_secure = false;
         bool m_httpOnly = false;
-        DateTime m_timeStamp = DateTime.Now;
+        DateTime m_timeStamp = DateTime.UtcNow;
```

Comment:
Why did this change?  The rest of the changes don't appear to leak out of the class, but this one does.  I also don't see how it relates to expiration times.

Reply:
You're right - it's separate than the expiration time.  This property is for the timestamp that the cookie was *issued*.

Both should be in terms of UTC, because timestamps always need to be in terms of UTC (or represented as a `DateTimeOffset`) - otherwise they can be misinterpreted.

Also, since `TimeStamp` and `Expired` are the two public APIs exposed, it's reasonable that they should both be of the same kind.  Otherwise subtracting them could give even further erroneous results.

And lastly, `DateTime.Now` incurs a higher perf cost than `DateTime.UtcNow` because it requires a time zone conversion internally.

Reply:
> Also, since TimeStamp and Expired are the two public APIs exposed, it's reasonable that they should both be of the same kind. Otherwise subtracting them could give even further erroneous results.

`Expired` is a bool. How do you "subtract them"?

> And lastly, DateTime.Now incurs a higher perf cost than DateTime.UtcNow because it requires a time zone conversion internally.

Yes.  But this is also changing observable behavior, as cookie.TimeStamp.Kind will now return Utc instead of Local.  There's no way that's going to break anyone?  Perf improvements are great, but not at the expense of compat if there's a real compat issue.